src/components/editor/property/PropertyResourceTemplate.jsx

```diff
@@ -21,12 +21,17 @@ class PropertyResourceTemplate extends Component {
     const existingOutputs = [...this.state.output]
 
     existingOutputs.push(<h4 key={shortid.generate()}>{this.props.resourceTemplate.resourceLabel}</h4>)
-    const result = this.populatePropertyTemplates()
+
+    const newReduxPath = [...this.props.reduxPath]
+
+    // Replace the generated id so that this is a new resource.
+    newReduxPath[newReduxPath.length - 2] = shortid.generate()
```

Comment:
I know I'm not looking at the whole here, but ... it is not obvious to me why we're working with an index of `newReduxPath.length - 2` ... could/should that be explained in the comment?

Reply:
Adding an example redux path so that it is clearer why `-2`.